docs/docs/animations/values.md

```diff
@@ -89,16 +89,9 @@ const Demo = () => {
 };
 ```
 
-## Canvas
+## Canvas Size
 
-The `useCanvas` hook returns a `size` value that updates every time the canvas size updates.
-On the first frame, the size is zero.
-
-:::caution
-
-`useCanvas` can only be used inside the Canvas element because it relies on context.
-
-:::
+The `onSize` property will assign a `size` to a Skia value.
```

Comment:
Maybe something like:

"The `onSize` property will update the provided Skia Value with the current size when the Canvas is resized."